### PR TITLE
Support prototext blocks without colon

### PR DESCRIPTION
**Pull request by jvmakine**

### CODE REVIEW

File: parser.go [hunk 1]
```diff
@@ -33,9 +33,20 @@ type Entry struct {
 type Option struct {
 	Pos lexer.Position
 
-	Name  string  `( "(" @("."? Ident { "." Ident }) ")" | @("."? Ident { "." Ident }) )`
-	Attr  *string `[ @("."? Ident { "." Ident }) ]`
-	Value *Value  `"=" @@`
+	Name  string     `( "(" @("."? Ident { "." Ident }) ")" | @("."? Ident { "." Ident }) )`
+	Attr  *string    `[ @("."? Ident { "." Ident }) ]`
+	Text  *ProtoText `"=" ( @@`
```
Comment (alecthomas) [inline parser.go]: Should this just be part of `Value`?

**Review by alecthomas — APPROVED** [review]

**Comment by alecthomas** [comment]
I will fix those lint errors...not sure why I didn't see them before.

File: parser.go (hunk 1 above)
Comment (jvmakine) [reply parser.go]: Hmm, how would the differentation here work if it was?
```
type ProtoTextField struct {
	Name  string     `@Ident `
	Text  *ProtoText `( ( ":"? @@ )`
	Value *Value     `  | ( ":" @@ ) )`
}
```

Comment (jvmakine) [reply parser.go]: i.e, `Value` requires `:` before it, while prototext does not necessarily need it

Comment (jvmakine) [reply parser.go]: Actually, I might see if I can refactor this a bit, potentially parsing values without `:` even though `protoc` fails for these

Comment (jvmakine) [reply parser.go]: Done, now the parser is slightly more permissive than `protoc` but I think that is a fine tradeoff for a nicer interface

**Pull request merged**